steps/release-notes-extractor/0.0.1/step.yml

```diff
@@ -0,0 +1,59 @@
+title: Release notes extractor
+summary: |
+  Easily extract release notes for your project's changelog
+description: |
+  With this step you can easily extract the last entry of your project's changelog to be used as release notes!
+website: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+source_code_url: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+support_url: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+published_at: 2019-11-29T19:52:50.120568739+02:00
+source:
+  git: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step.git
+  commit: 1894dcf471fca456bcf045eb18d5609ae212e4c3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
it seems your step is not using `git` or `wget` tools: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-dependencies

Reply:
You're right, I don't need it.
Weirdly enough, I didn't declare a `deps` section in the `step.yml` at all.
Any clue why it is putting this default `git` and `wget` dependency section when sharing the step through the CLI?

Reply:
This is the `step.yml` file I'm talking about
https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step/blob/develop/step.yml

Reply:
I updated the `steps.yml` with a `apt-get` dependency for `nodejs`, the pipeline fails and I'm not able to know why.

Reply:
@ashrafDoubleO7 so the `deps` section is generated by the step plugin, as an example. Might it would be better to uncomment this section, so new step developers, still see the example, but do not leave unused deps in the step.yml.

I will create a task to update the step plugin

Reply:
@ashrafDoubleO7 what is the error message?

Reply:
Thank you @godrei, I fixed it

Reply:
If this step is acceptable now, should I make another PR or is this one still valid?